docs/code-snippets/outlook-snippets.yaml

```diff
@@ -244,6 +306,28 @@ Office.AppointmentRead.attachments:
     }
 
     // Do something with outputString
+Office.AppointmentRead.body:
+  - |-
+    // This example gets the body of the message in plain text.
+    Office.context.mailbox.item.body.getAsync(
+        "text",
+        { asyncContext:"This is passed to the callback" },
+        function callback(result) {
+            // Do something with the result
+        });
+
+    //The following is an example of the result parameter passed to the callback function.
+    {
+        "value": "TEXT of whole body (including threads below)",
+        "status": "succeeded",
+        "asyncContext": "This is passed to the callback"
+    }
+Office.AppointmentRead.dateTimeCreated:
+  - |-
+    var dateTimeCreated = Office.context.mailbox.item.dateTimeCreated; 
```

Comment:
I wonder if we can't do slightly more with these examples. Like, maybe log something from the returned value?

Reply:
That said, if you think there's more value in just showing how to access the object through the OM, that's cool. I trust you.

Reply:
In some cases I have console.log statements instead of var declarations. I can switch to console.log instead.